extras/vagrantExample/src/main/vagrant/readme.md

```diff
@@ -41,6 +41,18 @@ Once Vagrant is installed, starting up the Rya-Example-Box is fairly straightfor
 
 1. Download the [Rya-Example-Box Vagrantfile] to the custom directory.  Note that it must be named `Vagrantfile` with no extension.
 
+1. Optionally copy built Rya libraries from your project into the vagrant directory.  If not, it will attempt to download from the public repository.  The following files can be copied or linked (ln -s file destination).
+
+        extras/indexingExample/target/rya.indexing.example-*-distribution.zip
+        extras/vagrantExample/target/rya.vagrant.example-*.jar
+        web/web.rya/target/web.rya.war
+
+The last one must be renamed to include the version, for example:  web.rya-4.0.1-incubating-SNAPSHOT.war
+
+1. Modify the Vagrant file to match the version on your files or in the repository.  It may already match the current version, but verify it since it is a pre-release guess.  This is around line 56.  For example:

```

Comment:
Change from "Modify the Vagrant file to match the version on your files or in the repository." to "Modify the Vagrant file so the RYA_EXAMPLE_VERSION matches the version in your files or in the public repository."